src/main/java/org/openrefine/extensions/commons/importer/RelatedCategoryFetcher.java

```diff
@@ -34,8 +34,18 @@ public RelatedCategoryFetcher(String apiUrl, Iterator<FileRecord> iteratorFileRe
         this.iteratorFileRecords = iteratorFileRecords;
     }
 
+    /*
+     * Utility method for testing api calls with variable number of titles
+     *
+     * @param limit
+     */
+    public void setApiLimit(int limit) {
+        API_TITLES_LIMIT = limit;
```

Comment:
This should not change the value of a static variable, but rather change the value of an attribute (not static). That attribute could be initialized with the static value as a default, and the rest of the code in this class should refer to the attribute (not static).